R/internalStandards.R

```diff
@@ -0,0 +1,63 @@
+#' @title Get internal standards
+#'
+#' @description
+#'
+#' `internalStandards` returns a table with metabolite standards available 
+#'     in commercial internal standard mixes
+#'
```

Comment:
would be nice if you could provide in the description a documentation of the individual columns, something like:

```r
#' - `"name"`: the name of the standard.
#' - `"formulaSalt"`: ...
```